### DISCUSSION TITLE
Separate HTO demultiplexed samples

### DISCUSSION THREAD

**Discussion by csweeney225** [satijalab/seurat#4787] [Q&A] [answered]
Hi,

I have 5' gene expression data, which has been multiplexed with hashtag oligonucleotides (HTOs), and I have successfully used Seurat to demultiplex and remove doublets. I would like to be able to analyse and visualise just a subset of the multiplexed samples (e.g. within each sample, there are 5 multiplexed samples and I would like to analyse just 2 or 3 of these component samples together).

Is there a way of either splitting into separate Seurat objects or just analysing a subset of the cells within the Seurat object?

Thanks,
Connor

**Comment by timoast** [comment, marked as answer]
See the `SplitObject` function: https://www.rdocumentation.org/packages/Seurat/versions/4.0.3/topics/SplitObject
